src/lib/crypto.h

```diff
@@ -63,6 +63,7 @@
 #include "packet-parse.h"
 #include "symmetric.h"
 #include "bn.h"
+#include <rekey/rnp_key_store.h>
```

Comment:
I have no idea why you include this header. Anyway, it is just style-remark, so, you can ignore it.